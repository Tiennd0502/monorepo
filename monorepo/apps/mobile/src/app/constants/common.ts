export const CONGRATS_IMAGE = 'https://i.ibb.co/nkf6hFr/congrats.png';
