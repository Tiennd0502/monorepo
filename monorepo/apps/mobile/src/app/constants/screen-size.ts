import { Dimensions } from 'react-native';

export const BASE_SCREEN_WIDTH = 412;
export const BASE_SCREEN_HEIGHT = 842;

export const SCREEN_WIDTH: number = Dimensions.get('window').width;
export const SCREEN_HEIGHT: number = Dimensions.get('window').height;

export const CONTENT_WIDTH: number = SCREEN_WIDTH - 40;
