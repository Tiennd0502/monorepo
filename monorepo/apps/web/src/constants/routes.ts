export const ROUTES = {
  HOME: '/',
  VERIFY_OTP: '/verify-otp',
  SIGN_UP: '/sign-up',
  LOGIN: '/login',
};
