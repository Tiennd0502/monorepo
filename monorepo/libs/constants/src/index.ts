export * from './apis';
export * from './message';
export * from './commons';
export * from './product';
export * from './category';
export * from './validation';
