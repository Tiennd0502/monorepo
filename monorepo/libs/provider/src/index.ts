export { default as TamaguiProvider } from './TamaguiProvider';
export { default as QueryClientProvider } from './QueryClientProvider';
