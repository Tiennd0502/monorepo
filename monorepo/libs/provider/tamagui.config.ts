import { createTamagui, createTokens } from 'tamagui';
import { config } from '@tamagui/config/v3';
import { themes } from '@tamagui/themes';

import {
  colorsMapping,
  radius,
  size,
  space,
  zIndex,
  defaultFont,
  headingFont,
} from '@monorepo/ui';

const tokens = {
  radius,
  size,
  space,
  zIndex,
};

const tamaguiConfig = createTamagui({
  ...config,
  defaultFont: 'body',
  themes,
  fonts: {
    body: defaultFont,
    heading: headingFont,
  },
  tokens: createTokens({
    ...tokens,
    color: colorsMapping,
  }),
});

type AppConfig = typeof config;

declare module 'tamagui' {
  type TamaguiCustomConfig = AppConfig;
}

export default tamaguiConfig;
