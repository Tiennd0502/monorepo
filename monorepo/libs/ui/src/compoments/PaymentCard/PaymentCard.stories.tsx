import { Meta, StoryObj } from '@storybook/react';
import { Stack } from 'tamagui';

import { PAYMENT_CARDS } from '@monorepo/constants';
import PaymentCard from '.';

const meta: Meta<typeof PaymentCard> = {
  title: 'Components/PaymentCard',
  component: PaymentCard,
  decorators: [
    (Story) => (
      <Stack width={450}>
        <Story />
      </Stack>
    ),
  ],
};

export default meta;

type Story = StoryObj<typeof PaymentCard>;

export const Default: Story = {
  args: {
    item: PAYMENT_CARDS[0],
  },
};

export const MasterCard: Story = {
  args: {
    item: PAYMENT_CARDS[1],
  },
};

export const VisaCard: Story = {
  args: {
    item: PAYMENT_CARDS[2],
  },
};
