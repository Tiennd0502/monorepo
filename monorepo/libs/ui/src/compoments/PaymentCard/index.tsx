import { memo } from 'react';
import { Image, Stack, XStack } from 'tamagui';

import { Card } from '@monorepo/types';
import { VISA_IMAGE, MASTER_IMAGE, BG_CARD } from '@monorepo/constants';

import Text from '../Text';

interface PaymentCardProps {
  item: Card;
  specialCharacter?: string;
  isDisabled?: boolean;
}

const PaymentCard = ({
  item: { id, name, expiryDate, isVisa = false, isMasterCard = false },
  specialCharacter = '',
  isDisabled = false,
}: PaymentCardProps) => {
  return (
    <Stack
      position="relative"
      minHeight="$45"
      width="100%"
      padding="$6"
      zIndex="$1"
      borderRadius="$2"
      overflow="hidden"
    >
      <Image
        resizeMode="stretch"
        position="absolute"
        top={0}
        left={0}
        right={0}
        bottom={0}
        zIndex="$2"
        minWidth="100%"
        minHeight="100%"
        source={{ uri: BG_CARD }}
      />
      <Stack zIndex="$3">
        <XStack gap="$2.5" paddingBottom="$6.5">
          {isMasterCard && (
            <Image width="$8" height="$6" source={{ uri: MASTER_IMAGE }} />
          )}
          {isVisa && (
            <Image width="$12.5" height="$4" source={{ uri: VISA_IMAGE }} />
          )}
        </XStack>
        <Text color="$secondary" fontSize="$5">
          {id}
        </Text>
        <XStack paddingTop="$8.5" justifyContent="space-between">
          <Stack>
            <Text fontSize="$3">Card Holder Name</Text>
            <Text color="$secondary">
              {specialCharacter ? specialCharacter.repeat(6) : name}
            </Text>
          </Stack>
          <Stack>
            <Text fontSize="$3">Expiry Date</Text>
            <Text color="$secondary">
              {specialCharacter
                ? `${specialCharacter.repeat(2)}/${specialCharacter.repeat(2)}`
                : expiryDate}
            </Text>
          </Stack>
        </XStack>
      </Stack>
      {isDisabled && (
        <Stack
          position="absolute"
          top={0}
          left={0}
          right={0}
          bottom={0}
          zIndex="$4"
          backgroundColor="$backgroundLayer"
        />
      )}
    </Stack>
  );
};

export default memo(PaymentCard);
