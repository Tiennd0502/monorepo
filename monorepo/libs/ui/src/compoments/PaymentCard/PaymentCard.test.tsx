import {render} from '@testing-library/react-native';

import { TamaguiProvider } from '@monorepo/provider';
import { PAYMENT_CARDS } from '@monorepo/constants';
import PaymentCard from '.';

const onEdit = jest.fn();

const baseProps = {
  item: PAYMENT_CARDS[0],
  onEdit,
};

const setup = (moreProps = {}) => {
  const props = {
    ...baseProps,
    ...moreProps,
  };
  
  return render(
    <TamaguiProvider>
      <PaymentCard {...props} />
    </TamaguiProvider>
  );
};

describe('PaymentCard', () => {

  it('should render PaymentCard component', () => {
    const container = setup();

    expect(container).toMatchSnapshot();
  });

  it('should render PaymentCard component with isMasterCard', () => {
    const container = setup({item: PAYMENT_CARDS[1]});

    expect(container).toMatchSnapshot();
  });

  it('should render PaymentCard component with isVisaCard', () => {
    const container =  setup({item: PAYMENT_CARDS[2]});

    expect(container).toMatchSnapshot();
  });

  it('should render PaymentCard component has special character', () => {
    const container =  setup({ specialCharacter: 'X' });

    expect(container).toMatchSnapshot();
  });

  it('should render PaymentCard component disabled', () => {
    const container = setup({ isDisabled: true});

    expect(container).toMatchSnapshot();
  });
});
