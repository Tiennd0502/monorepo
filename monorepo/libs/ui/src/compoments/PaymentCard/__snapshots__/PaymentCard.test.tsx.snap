// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PaymentCard should render PaymentCard component 1`] = `
<View
  style={
    {
      "borderBottomLeftRadius": 8,
      "borderBottomRightRadius": 8,
      "borderTopLeftRadius": 8,
      "borderTopRightRadius": 8,
      "minHeight": 180,
      "overflow": "hidden",
      "paddingBottom": 24,
      "paddingLeft": 24,
      "paddingRight": 24,
      "paddingTop": 24,
      "position": "relative",
      "width": "100%",
      "zIndex": 100,
    }
  }
>
  <Image
    resizeMode="stretch"
    source={
      {
        "uri": "https://i.ibb.co/NTwMB09/bg-card.png",
      }
    }
    style={
      {
        "bottom": 0,
        "left": 0,
        "minHeight": "100%",
        "minWidth": "100%",
        "position": "absolute",
        "right": 0,
        "top": 0,
        "zIndex": 200,
      }
    }
  />
  <View
    style={
      {
        "zIndex": 300,
      }
    }
  >
    <View
      style={
        {
          "flexDirection": "row",
          "gap": 10,
          "paddingBottom": 26,
        }
      }
    >
      <Image
        source={
          {
            "uri": "https://i.ibb.co/S0tfTB1/master-card.png",
          }
        }
        style={
          {
            "height": 24,
            "width": 32,
          }
        }
      />
      <Image
        source={
          {
            "uri": "https://i.ibb.co/4gV41yh/visa.png",
          }
        }
        style={
          {
            "height": 16,
            "width": 50,
          }
        }
      />
    </View>
    <Text
      allowFontScaling={false}
      style={
        {
          "color": "#fff",
          "fontFamily": "NunitoSans_7pt-Medium",
          "fontSize": 20,
          "lineHeight": 20,
        }
      }
      suppressHighlighting={true}
    >
      * * * *  * * * *  * * * *  1237
    </Text>
    <View
      style={
        {
          "flexDirection": "row",
          "justifyContent": "space-between",
          "paddingTop": 34,
        }
      }
    >
      <View>
        <Text
          allowFontScaling={false}
          style={
            {
              "color": "#999",
              "fontFamily": "NunitoSans_7pt-Medium",
              "fontSize": 12,
              "lineHeight": 20,
            }
          }
          suppressHighlighting={true}
        >
          Card Holder Name
        </Text>
        <Text
          allowFontScaling={false}
          style={
            {
              "color": "#fff",
              "fontFamily": "NunitoSans_7pt-Medium",
              "fontSize": 14,
              "lineHeight": 20,
            }
          }
          suppressHighlighting={true}
        >
          Jennyfer Doe
        </Text>
      </View>
      <View>
        <Text
          allowFontScaling={false}
          style={
            {
              "color": "#999",
              "fontFamily": "NunitoSans_7pt-Medium",
              "fontSize": 12,
              "lineHeight": 20,
            }
          }
          suppressHighlighting={true}
        >
          Expiry Date
        </Text>
        <Text
          allowFontScaling={false}
          style={
            {
              "color": "#fff",
              "fontFamily": "NunitoSans_7pt-Medium",
              "fontSize": 14,
              "lineHeight": 20,
            }
          }
          suppressHighlighting={true}
        >
          05/23
        </Text>
      </View>
    </View>
  </View>
</View>
`;

exports[`PaymentCard should render PaymentCard component disabled 1`] = `
<View
  style={
    {
      "borderBottomLeftRadius": 8,
      "borderBottomRightRadius": 8,
      "borderTopLeftRadius": 8,
      "borderTopRightRadius": 8,
      "minHeight": 180,
      "overflow": "hidden",
      "paddingBottom": 24,
      "paddingLeft": 24,
      "paddingRight": 24,
      "paddingTop": 24,
      "position": "relative",
      "width": "100%",
      "zIndex": 100,
    }
  }
>
  <Image
    resizeMode="stretch"
    source={
      {
        "uri": "https://i.ibb.co/NTwMB09/bg-card.png",
      }
    }
    style={
      {
        "bottom": 0,
        "left": 0,
        "minHeight": "100%",
        "minWidth": "100%",
        "position": "absolute",
        "right": 0,
        "top": 0,
        "zIndex": 200,
      }
    }
  />
  <View
    style={
      {
        "zIndex": 300,
      }
    }
  >
    <View
      style={
        {
          "flexDirection": "row",
          "gap": 10,
          "paddingBottom": 26,
        }
      }
    >
      <Image
        source={
          {
            "uri": "https://i.ibb.co/S0tfTB1/master-card.png",
          }
        }
        style={
          {
            "height": 24,
            "width": 32,
          }
        }
      />
      <Image
        source={
          {
            "uri": "https://i.ibb.co/4gV41yh/visa.png",
          }
        }
        style={
          {
            "height": 16,
            "width": 50,
          }
        }
      />
    </View>
    <Text
      allowFontScaling={false}
      style={
        {
          "color": "#fff",
          "fontFamily": "NunitoSans_7pt-Medium",
          "fontSize": 20,
          "lineHeight": 20,
        }
      }
      suppressHighlighting={true}
    >
      * * * *  * * * *  * * * *  1237
    </Text>
    <View
      style={
        {
          "flexDirection": "row",
          "justifyContent": "space-between",
          "paddingTop": 34,
        }
      }
    >
      <View>
        <Text
          allowFontScaling={false}
          style={
            {
              "color": "#999",
              "fontFamily": "NunitoSans_7pt-Medium",
              "fontSize": 12,
              "lineHeight": 20,
            }
          }
          suppressHighlighting={true}
        >
          Card Holder Name
        </Text>
        <Text
          allowFontScaling={false}
          style={
            {
              "color": "#fff",
              "fontFamily": "NunitoSans_7pt-Medium",
              "fontSize": 14,
              "lineHeight": 20,
            }
          }
          suppressHighlighting={true}
        >
          Jennyfer Doe
        </Text>
      </View>
      <View>
        <Text
          allowFontScaling={false}
          style={
            {
              "color": "#999",
              "fontFamily": "NunitoSans_7pt-Medium",
              "fontSize": 12,
              "lineHeight": 20,
            }
          }
          suppressHighlighting={true}
        >
          Expiry Date
        </Text>
        <Text
          allowFontScaling={false}
          style={
            {
              "color": "#fff",
              "fontFamily": "NunitoSans_7pt-Medium",
              "fontSize": 14,
              "lineHeight": 20,
            }
          }
          suppressHighlighting={true}
        >
          05/23
        </Text>
      </View>
    </View>
  </View>
  <View
    style={
      {
        "backgroundColor": "rgba(153, 153, 153, 0.6)",
        "bottom": 0,
        "left": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
        "zIndex": 400,
      }
    }
  />
</View>
`;

exports[`PaymentCard should render PaymentCard component has special character 1`] = `
<View
  style={
    {
      "borderBottomLeftRadius": 8,
      "borderBottomRightRadius": 8,
      "borderTopLeftRadius": 8,
      "borderTopRightRadius": 8,
      "minHeight": 180,
      "overflow": "hidden",
      "paddingBottom": 24,
      "paddingLeft": 24,
      "paddingRight": 24,
      "paddingTop": 24,
      "position": "relative",
      "width": "100%",
      "zIndex": 100,
    }
  }
>
  <Image
    resizeMode="stretch"
    source={
      {
        "uri": "https://i.ibb.co/NTwMB09/bg-card.png",
      }
    }
    style={
      {
        "bottom": 0,
        "left": 0,
        "minHeight": "100%",
        "minWidth": "100%",
        "position": "absolute",
        "right": 0,
        "top": 0,
        "zIndex": 200,
      }
    }
  />
  <View
    style={
      {
        "zIndex": 300,
      }
    }
  >
    <View
      style={
        {
          "flexDirection": "row",
          "gap": 10,
          "paddingBottom": 26,
        }
      }
    >
      <Image
        source={
          {
            "uri": "https://i.ibb.co/S0tfTB1/master-card.png",
          }
        }
        style={
          {
            "height": 24,
            "width": 32,
          }
        }
      />
      <Image
        source={
          {
            "uri": "https://i.ibb.co/4gV41yh/visa.png",
          }
        }
        style={
          {
            "height": 16,
            "width": 50,
          }
        }
      />
    </View>
    <Text
      allowFontScaling={false}
      style={
        {
          "color": "#fff",
          "fontFamily": "NunitoSans_7pt-Medium",
          "fontSize": 20,
          "lineHeight": 20,
        }
      }
      suppressHighlighting={true}
    >
      * * * *  * * * *  * * * *  1237
    </Text>
    <View
      style={
        {
          "flexDirection": "row",
          "justifyContent": "space-between",
          "paddingTop": 34,
        }
      }
    >
      <View>
        <Text
          allowFontScaling={false}
          style={
            {
              "color": "#999",
              "fontFamily": "NunitoSans_7pt-Medium",
              "fontSize": 12,
              "lineHeight": 20,
            }
          }
          suppressHighlighting={true}
        >
          Card Holder Name
        </Text>
        <Text
          allowFontScaling={false}
          style={
            {
              "color": "#fff",
              "fontFamily": "NunitoSans_7pt-Medium",
              "fontSize": 14,
              "lineHeight": 20,
            }
          }
          suppressHighlighting={true}
        >
          XXXXXX
        </Text>
      </View>
      <View>
        <Text
          allowFontScaling={false}
          style={
            {
              "color": "#999",
              "fontFamily": "NunitoSans_7pt-Medium",
              "fontSize": 12,
              "lineHeight": 20,
            }
          }
          suppressHighlighting={true}
        >
          Expiry Date
        </Text>
        <Text
          allowFontScaling={false}
          style={
            {
              "color": "#fff",
              "fontFamily": "NunitoSans_7pt-Medium",
              "fontSize": 14,
              "lineHeight": 20,
            }
          }
          suppressHighlighting={true}
        >
          XX/XX
        </Text>
      </View>
    </View>
  </View>
</View>
`;

exports[`PaymentCard should render PaymentCard component with isMasterCard 1`] = `
<View
  style={
    {
      "borderBottomLeftRadius": 8,
      "borderBottomRightRadius": 8,
      "borderTopLeftRadius": 8,
      "borderTopRightRadius": 8,
      "minHeight": 180,
      "overflow": "hidden",
      "paddingBottom": 24,
      "paddingLeft": 24,
      "paddingRight": 24,
      "paddingTop": 24,
      "position": "relative",
      "width": "100%",
      "zIndex": 100,
    }
  }
>
  <Image
    resizeMode="stretch"
    source={
      {
        "uri": "https://i.ibb.co/NTwMB09/bg-card.png",
      }
    }
    style={
      {
        "bottom": 0,
        "left": 0,
        "minHeight": "100%",
        "minWidth": "100%",
        "position": "absolute",
        "right": 0,
        "top": 0,
        "zIndex": 200,
      }
    }
  />
  <View
    style={
      {
        "zIndex": 300,
      }
    }
  >
    <View
      style={
        {
          "flexDirection": "row",
          "gap": 10,
          "paddingBottom": 26,
        }
      }
    >
      <Image
        source={
          {
            "uri": "https://i.ibb.co/S0tfTB1/master-card.png",
          }
        }
        style={
          {
            "height": 24,
            "width": 32,
          }
        }
      />
    </View>
    <Text
      allowFontScaling={false}
      style={
        {
          "color": "#fff",
          "fontFamily": "NunitoSans_7pt-Medium",
          "fontSize": 20,
          "lineHeight": 20,
        }
      }
      suppressHighlighting={true}
    >
      * * * *  * * * *  * * * *  1235
    </Text>
    <View
      style={
        {
          "flexDirection": "row",
          "justifyContent": "space-between",
          "paddingTop": 34,
        }
      }
    >
      <View>
        <Text
          allowFontScaling={false}
          style={
            {
              "color": "#999",
              "fontFamily": "NunitoSans_7pt-Medium",
              "fontSize": 12,
              "lineHeight": 20,
            }
          }
          suppressHighlighting={true}
        >
          Card Holder Name
        </Text>
        <Text
          allowFontScaling={false}
          style={
            {
              "color": "#fff",
              "fontFamily": "NunitoSans_7pt-Medium",
              "fontSize": 14,
              "lineHeight": 20,
            }
          }
          suppressHighlighting={true}
        >
          Jennyfer Doe
        </Text>
      </View>
      <View>
        <Text
          allowFontScaling={false}
          style={
            {
              "color": "#999",
              "fontFamily": "NunitoSans_7pt-Medium",
              "fontSize": 12,
              "lineHeight": 20,
            }
          }
          suppressHighlighting={true}
        >
          Expiry Date
        </Text>
        <Text
          allowFontScaling={false}
          style={
            {
              "color": "#fff",
              "fontFamily": "NunitoSans_7pt-Medium",
              "fontSize": 14,
              "lineHeight": 20,
            }
          }
          suppressHighlighting={true}
        >
          05/23
        </Text>
      </View>
    </View>
  </View>
</View>
`;

exports[`PaymentCard should render PaymentCard component with isVisaCard 1`] = `
<View
  style={
    {
      "borderBottomLeftRadius": 8,
      "borderBottomRightRadius": 8,
      "borderTopLeftRadius": 8,
      "borderTopRightRadius": 8,
      "minHeight": 180,
      "overflow": "hidden",
      "paddingBottom": 24,
      "paddingLeft": 24,
      "paddingRight": 24,
      "paddingTop": 24,
      "position": "relative",
      "width": "100%",
      "zIndex": 100,
    }
  }
>
  <Image
    resizeMode="stretch"
    source={
      {
        "uri": "https://i.ibb.co/NTwMB09/bg-card.png",
      }
    }
    style={
      {
        "bottom": 0,
        "left": 0,
        "minHeight": "100%",
        "minWidth": "100%",
        "position": "absolute",
        "right": 0,
        "top": 0,
        "zIndex": 200,
      }
    }
  />
  <View
    style={
      {
        "zIndex": 300,
      }
    }
  >
    <View
      style={
        {
          "flexDirection": "row",
          "gap": 10,
          "paddingBottom": 26,
        }
      }
    >
      <Image
        source={
          {
            "uri": "https://i.ibb.co/4gV41yh/visa.png",
          }
        }
        style={
          {
            "height": 16,
            "width": 50,
          }
        }
      />
    </View>
    <Text
      allowFontScaling={false}
      style={
        {
          "color": "#fff",
          "fontFamily": "NunitoSans_7pt-Medium",
          "fontSize": 20,
          "lineHeight": 20,
        }
      }
      suppressHighlighting={true}
    >
      * * * *  * * * *  * * * *  1236
    </Text>
    <View
      style={
        {
          "flexDirection": "row",
          "justifyContent": "space-between",
          "paddingTop": 34,
        }
      }
    >
      <View>
        <Text
          allowFontScaling={false}
          style={
            {
              "color": "#999",
              "fontFamily": "NunitoSans_7pt-Medium",
              "fontSize": 12,
              "lineHeight": 20,
            }
          }
          suppressHighlighting={true}
        >
          Card Holder Name
        </Text>
        <Text
          allowFontScaling={false}
          style={
            {
              "color": "#fff",
              "fontFamily": "NunitoSans_7pt-Medium",
              "fontSize": 14,
              "lineHeight": 20,
            }
          }
          suppressHighlighting={true}
        >
          Jennyfer Doe
        </Text>
      </View>
      <View>
        <Text
          allowFontScaling={false}
          style={
            {
              "color": "#999",
              "fontFamily": "NunitoSans_7pt-Medium",
              "fontSize": 12,
              "lineHeight": 20,
            }
          }
          suppressHighlighting={true}
        >
          Expiry Date
        </Text>
        <Text
          allowFontScaling={false}
          style={
            {
              "color": "#fff",
              "fontFamily": "NunitoSans_7pt-Medium",
              "fontSize": 14,
              "lineHeight": 20,
            }
          }
          suppressHighlighting={true}
        >
          05/23
        </Text>
      </View>
    </View>
  </View>
</View>
`;
